### DISCUSSION TITLE
Can we have a Ignore Scenario tag? or even a Ignore Test tag? maybe both?

### DISCUSSION THREAD

**Discussion by eliezercazares** [artilleryio/artillery#1466] [Artillery]
In Cucumber i.e. or with Java/.NET annotations you can specify tags for methods, tests, etc.. then you can ignore or run files and scenarios based on that tag.

i.e.

```
- tag: ignore
config:
...
```

**Comment by hassy** [comment]
hi @eliezercazares 👋 

what does your workflow look like? e.g. in what conditions would you choose to ignore some scenarios?

**Reply by eliezercazares** [reply]
> hi @eliezercazares 👋
> 
> what does your workflow look like? e.g. in what conditions would you choose to ignore some scenarios?

Hello hassy!
Hope you're doing well!

Well, it might be related to this item actually:
https://github.com/artilleryio/artillery/issues/167

But also, as far as I have learned (I am still creating my first scenarios) we can have multiple scenarios in one single .yaml file in artillery.

From an online blog:
```
 "But you can also define multiple scenarios to simulate users doing different things at your application at the same time.

Let´s say you want to simulate users searching and buying products at the same time. You can create two different scenarios to test both flows. Note that each user will only execute one of the scenarios.

The scenario supports a "weight" property which you can use to indicate how likely it is that scenario to be picked by a particular user.

By default, each scenario will have the same probability. If you define a scenario with weight 2, that scenario is twice as likely to be picked as the one with 1 (which is the default value)."
```

So, let's say that I have a chat application I am load testing, and I have batches of users doing message posts and replies, other batches doing multimedia sharing, other batches setting their profile, etc...
For this I can create 3 or more scenarios in the YAML.

But sometimes I would be interested in testing only the multimedia part and the profile only.

Let me know if I am missing out details.


Cucumber tests i.e. can be ignored with a tag at the beginning of the file, that will tell the engine to ignore it all.

Or you can have the ignore tags within each scenario, before each.
